Dockerfile

```diff
@@ -59,12 +59,59 @@ FROM --platform=$BUILDPLATFORM alpine AS releaser
 WORKDIR /work
 ARG TARGETPLATFORM
 RUN --mount=from=binaries \
-  --mount=type=bind,source=/tmp/.version,target=/tmp/.version,from=buildx-version \
+  --mount=type=bind,source=/tmp/.version,target=/tmp/.version,from=version \
   mkdir -p /out && cp buildx* "/out/buildx-$(cat /tmp/.version).$(echo $TARGETPLATFORM | sed 's/\//-/g')$(ls buildx* | sed -e 's/^buildx//')"
 
 FROM scratch AS release
 COPY --from=releaser /out/ /
 
+# Deb
+FROM --platform=$BUILDPLATFORM debian:bullseye-slim AS build-deb
+RUN apt-get update && apt-get install -y debhelper dh-make
+WORKDIR /build
+COPY --from=xx / /
+COPY --from=build /usr/bin/buildx ./docker-buildx-standalone/usr/bin/
+COPY --from=build /usr/bin/buildx ./docker-buildx-plugin/usr/libexec/docker/cli-plugins/docker-buildx
```

Comment:
I don't think we need two packages. We can just put the binary at `/usr/libexec/docker/cli-plugins/docker-buildx` and symlink it to `/usr/bin/buildx`.